demisto_sdk/commands/validate/validators/IN_validators/IN119_feed_integration_from_version.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from packaging.version import Version
+
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    FixResult,
+    ValidationResult,
+)
+
+ContentTypes = Integration
+
+
+class FeedIntegrationFromVersionValidator(BaseValidator[ContentTypes]):
```

Comment:
why can this be IsFromVersionSufficientValidator ?
If behavior is only slightly different lets try to use inheritance